### DISCUSSION TITLE
Notes for SRR subsystem

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/sys-581-group-project#31] [Ideas]
Useful sources:
- math for power required vs transmission range relationship: https://www.electronicdesign.com/technologies/communications/article/21796484/understanding-wireless-range-calculations

**Comment by Victor-M-Giraldo** [comment]
Relevant to SRR
- https://www.nasa.gov/smallsat-institute/sst-soa/soa-communications/
- https://www.digi.com/blog/post/long-range-vs-short-range-wireless-communications
